sections/03_categorize.md

```diff
@@ -520,33 +520,46 @@ and Roth [@doi:10.1561/0400000042] demonstrate the ability to expose individual
-model from such attacks.
-
-There are approaches to protect models, but they pose their own challenges.
-Training in a differentially private manner provides a limited guarantee that an
-algorithm's output will be equally likely to occur regardless of the
-participation of any one individual. The limit is determined by a single
-parameter which provides a quantification of privacy. Simmons et al.
-[@doi:10.1016/j.cels.2016.04.013] present the ability to perform genome-wide
-association studies (GWASs) in a differentially private manner, and Abadi et al.
-[@arxiv:1607.00133] show the ability to train deep learning classifiers under
-the differential privacy framework. Federated learning
-[@url:http://proceedings.mlr.press/v54/mcmahan17a.html] and secure aggregations
-[@url:https://eprint.iacr.org/2017/281.pdf @tag:Papernot2017_pate] are
-complementary approaches that reinforce differential privacy. Both aim to
-maintain privacy by training deep learning models from decentralized data
-sources such as personal mobile devices without transferring actual training
-instances. This is becoming of increasing importance with the rapid growth of
-mobile health applications. However, the training process in these approaches
-places constraints on the algorithms used and can make fitting a model
-substantially more challenging. In our own experience, it can be trivial to
-train a model without differential privacy, but quite difficult to train one
-within the differential privacy framework. The problem can be particularly
-pronounced with small sample sizes.
+this case, whether a person's record was present. To protect against these
+attacks, Simmons et al. [@doi:10.1016/j.cels.2016.04.013] developed the ability
+to perform genome-wide association studies (GWASs) in a differentially private
+manner, and Abadi et al. [@arxiv:1607.00133] show the ability to train deep
+learning classifiers under the differential privacy framework.
+
+These attacks also present a potential hazard for approaches that aim to
+generate data. Choi et al. propose generative adversarial neural networks (GANs)
+as a tool to make sharable EHR data [@arxiv:1703.06490v1], and Hyland et al.
+[@arxiv:1706.02633v1] showed that recurrent GANs could be used for time series
+data. However, in both cases the authors did not take steps to protect the model
+from such attacks. There are approaches to protect models, but they pose their
+own challenges. Training in a differentially private manner provides a limited
+guarantee that an algorithm's output will be equally likely to occur regardless
+of the participation of any one individual. The limit is determined by
+parameters which provide a quantification of privacy. Beaulieu-Jones et al.
+demonstrated the ability to generate data that preserved properties of the
+SPRINT clinical trial with GANs under the differential privacy framework
+[@doi:10.1101/159756]. In subsequent, concurrent revisions Hyland et al.
```

Comment:
As an outsider, this does seem like it is getting too detailed about who did what first.  I think the main point is that paper A introduced one important feature and paper B introduced another.  Revisions shows that these ideas are compatible (cite both again), which is likely the path forward.

It seems confusing to introduce version 1 of Hyland (Esteban?) et al. at the top of the paragraph and its susceptibility to attacks and then mention it again when differential privacy has been added.

Reply:
Sounds good. I will revise this 👍